tests/unit/util/util-test.js

```diff
@@ -405,4 +405,11 @@ QUnit.test('[ID19] Test arrayCopy1', function (assert) {
 	target = [6,7,8,9,10];
 	$.ig.util.arrayCopy1(arr, 0, target, 1, 4);
 	assert.deepEqual(target, [6,1,2,3,4], "Copy content to the end of another array");
-});
\ No newline at end of file
+});
+
+QUnit.test('[ID20] Test OADate', function (assert) {
+	assert.expect(1);
+
+	var dt = new Date(2000, 1, 1);
```

Comment:
@igandrews Maybe write a few more asserts with a larger variety of dates.

Reply:
I'll add more tests. How come grunt verify didn't catch the errors with the newline in the test?

Reply:
I think tests are excluded from linting.

Reply:
Shouldn't it verify everything so one doesn't need to rely on waiting for the PR results?

Reply:
It totally should... The thing is, if I turn it on right now, there will be A LOT of lint errors :S